utils/src/main/kotlin/DiskCache.kt

```diff
@@ -113,7 +113,7 @@ class DiskCache(
         try {
```

Comment:
Please add a rationale *why* we do this (similar to the previous commit, because the new name is more speaking).

Reply:
Done.